packages/components/checkbox/src/checkbox-group.ts

```diff
@@ -236,7 +236,7 @@ export class CheckboxGroup<T = any> extends FormControlMixin(LitElement) {
   #updateValidity(): void {
     this.internals.setValidity(
       { valueMissing: this.required && !this.boxes?.some(box => box.checked) },
-      msg('Please check at least one option.')
+      msg('Please check at least one option.', { id: 'sl.checkboxGroup.validation.required' })
```

Comment:
Perhaps use `sl.checkboxGroup.validation.valueMissing` since that matches with the validity API.